### ISSUE TITLE
Icon submission: Buenbit

### ISSUE THREAD

**Issue by sebastianserrani** [krisu5/aegis-icons#564]
### Icon type

Logo based

### URL of the site or app

https://www.buenbit.com

### Logo / icon info

Image traced of https://assets-global.website-files.com/617c7ed623b070e01c889509/617ff20f05064990d7ae786e_favicon.png

### Background color info

Background color is accent color frow web page.

### Upload icon(s)

[Buenbit.zip](https://github.com/aegis-icons/aegis-icons/files/8400621/Buenbit.zip)

### Confirmation

- [X] I have read and **carefully** followed [CONTRIBUTING document](https://github.com/aegis-icons/aegis-icons/blob/master/CONTRIBUTING.md)

**Comment by krisu5** [comment]
> Image traced of https://assets-global.website-files.com/617c7ed623b070e01c889509/617ff20f05064990d7ae786e_favicon.png

With what image tracing software? Also, I highly don't recommend image tracing really low resolution images.

Use actual logo, not favicon: https://assets-global.website-files.com/617c7ed623b070e01c889509/617c7ed623b0700e0a88952a_logo_buen-bit-SVG.svg

> Background color is accent color frow web page.

From where? You need to either:

- [Upload](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/attaching-files) screenshot of the area where you got the color **and** give URL of the page
- Give the HTML element name (eg. `#footer`) **and** URL of the page.

_And just a small note for relating to other issues: image tracing means making vector out of raster image (eg. JPG to SVG), so sentence "image traced of" with link to SVG doesn't make sense._

**Comment by sebastianserrani** [comment]
Sorry! was in a hurry and made a mess. Originally, I used the svg logo you linked and fixed it manually as I didn't found any other proper "isotype" on their web site. I've now found the actual logo (https://app.buenbit.com/static/media/icon-logo-mobile.6b3a693c.svg) and used it to make a new svg.
As for the background color, I picked the class "highlight_blue-sky" (#1a9cff), as they use it as the accent/primary color on the landing page (https://www.buenbit.com/es-ar); on this new attached svg, I used the background color of the logo recently found (#00317a).
Hope this helps. Greeting
[Buenbit.zip](https://github.com/aegis-icons/aegis-icons/files/8411748/Buenbit.zip)
s
